src/components/Footer.js

```diff
@@ -107,15 +107,22 @@ const Footer = () => {
 					</BottomSection>
 				</FooterBottom>
 			</FooterWrapper>
-		</>
+		</Container>
 	);
 };
 
 export default Footer;
 
 // styled-components
-const FooterWrapper = styled.div`
+const Container = styled.div`
+	width: 100%;
 	background-color: #f7f7f7;
+`;
+
+const FooterWrapper = styled.div`
+	width: 100%;
+	max-width: 1800px;
+	margin: 0 auto;
 	font-size: 14px;
 	padding-bottom: 3px;
 `;
```

Comment:
`<LeftDiv>` 와 `<RightDiv>` 는 하나의 스타일컴포넌트로 합치셔도 좋을거같습니다~ Div라는 이름을 써도 되는지 모르겠는데, 이름 짓기가 어려울땐 Styled를 붙혀서 StyledDiv 또는 FlexDiv 해도 좋을거같습니다~ 여기서 확장되어 각 스타일링을 추가 해줘야할때는 상속시켜서하는 방법? 고려해봐도 좋구요~

Reply:
밑에 보다보니 위에 하나의 div로 합치고 난 후 `RightOption`에 extending styles를 적용해볼수있을거같아요!

```suggestion
const RightOption = styled(위에지어주는컴포넌트이름)`
	margin-right: 30px;
`;
```

Reply:
192번~200번 줄 부분 참고하시면 됩니당.. 선택이 안되는 부분이라 위에서 댓글 드렸더니 일부분이 지워졌네요 ㅜ 혹시 댓글 이해가 안되시면 질문 주세요!